### PR TITLE
feat(web3): get peers from registry 

### PR DESCRIPTION
**Pull request by Mitsichury**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
...ess","name":"account","type":"address"}],"name":"Stake","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"internalType":"address","name":"account","type":"address"}],"name":"Unstake","type":"event"},{"inputs":[{"internalType":"address","name":"staker","type":"address"}],"name":"balanceOf","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"},{"inputs":[],"name":"getURLs","outputs":[{"internalType":"string[]","name":"urls","type":"string[]"}],"stateMutability":"view","type":"function"},{"inputs":[],"name":"obligationCost","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"},{"inputs":[{"internalType":"string","name":"_url","type":"string"}],"name":"setURL","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"address","name":"","type":"address"}],"name":"stakerURLs","outputs":[{"internalType":"string","name":"","type":"string"}],"stateMutability":"view","type":"function"},{"inputs":[],"name":"stakingToken","outputs":[{"internalType":"contract IERC20","name":"","type":"address"}],"stateMutability":"view","type":"function"}]'
```
Comment (Mitsichury) [inline .env.example:10]: current goerli indexer registry is 0xC32a3c867aBAd28d977e1724f92D9684fF3d2976
you can also grab latest indexer registry addresses from @airswap/indexers/deploys.js

File: README.md [hunk 2]
```diff
@@ -2,28 +2,70 @@
...
+REGISTRY="http://localhost:4000"  
+</pre>
 
+Find ethernet card names: ifconfig/ipconfig/ip address and paste it in LOCAL_INTERFACES
+
+In a first terminal
 > $ yarn registry
 
-Finally in antoher terminal
+Then in anoter one:
```
Comment (dmosites) [inline README.md:63]: "another"

File: src/client/__tests__/Web3RegistryClient.test.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import { ContractInterface } from 'ethers';
+import { Web3RegistryClient } from './../Web3RegistryClient';
+import { ethers } from "ethers";
+
+jest.mock('ethers');
+const mockedEther = ethers as jest.Mocked<typeof ethers>;
+
+describe("Web3RegistryClient", () => {
+    const apiKey = "apikey";
+    const registryAddress = "registryAddres";
```
Comment (dmosites) [inline src/client/__tests__/Web3RegistryClient.test.ts:11]: "registryAddress"

**Review by dmosites — CHANGES_REQUESTED** [review]
Looks sane to me. Think we should remove the existing registry server and client to remove any confusion about server discovery.

File: README.md (hunk 2 above)
Comment (Mitsichury) [reply README.md]: thx :)

**Review by dmosites — APPROVED** [review]

**Pull request merged**